openedx/core/djangoapps/user_authn/views/login.py

```diff
@@ -356,7 +356,7 @@ def login_user(request):
                 _enforce_password_policy_compliance(request, possibly_authenticated_user)
 
         if possibly_authenticated_user is None or not possibly_authenticated_user.is_active:
-            _handle_failed_authentication(email_user)
+            _handle_failed_authentication(email_user, possibly_authenticated_user)
```

Comment:
As someone who participated in debugging this issue ina late evening, I can say first-hand that this code wasn't easily readable - primarily due to the current values of the variable names.   While we're fixing this issue and reviewing this code, can we update the variable names so they are more accurate?  Specifically, can we change `email_user` to just `user` if that's what it really is?

Also, when passing values to a boolean argument, it's more readable if you specify the name of the boolean argument in the call.   So, this call could be:

`_handle_failed_authentication(user, is_user_authenticated=possibly_authenticated_user is not None)`

Reply:
I can go through the code and see where the re-factoring can be applied.